src/CG/CG_elem.cpp

```diff
@@ -2113,6 +2113,829 @@ bool intersectSegmentPolygon( array3D const &P0, array3D const &P1, std::size_t
     return false;
 }
 
+/*!
+ * Check if a pixel is crossing a given plane and fully calculate the intersection (a segment) with 
+ * edges, eventually. A pixel is defined as a planar rectangular, vertical to a cartesian axis.
```

Comment:
As far as I know, there is no requirement for the normal of the plane in which the pixel lies to be aligned with one of the Cartesian axes.

Reply:
Done. Now this function handles pixels of an arbitrary orientation.